frontend/src/pages/org/settings.ts

```diff
@@ -301,6 +309,44 @@ export class OrgSettings extends LiteElement {
     `;
   }
 
+  private renderBilling() {
+    return html`<div class="rounded border">
+      <form @submit=${this.onOrgBillingSubmit}>
+        <div class="grid grid-cols-5 gap-x-4 p-4">
```

Comment:
This needs an extra wrapper so the footer section works I think.  The footer has outer padding applied but shouldn't.

<img width="953" alt="Screenshot 2023-10-17 at 3 58 17 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/5672810/7aa412d0-3f1a-4afe-926c-101bea3446e8">

See General tab for reference.
<img width="964" alt="Screenshot 2023-10-17 at 3 59 36 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/5672810/4008e8a7-6db3-43ac-926d-830c86a14e58">

Reply:
Fixed!

<img width="1161" alt="Screen Shot 2023-10-17 at 5 05 16 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/6758804/fcf7dab3-e3b4-4011-95f2-e756adb95cd7">